### DISCUSSION TITLE
npx nuxi init error

### DISCUSSION THREAD

**Discussion by intermundos** [nuxt/nuxt#17206] [Questions]
### Environment

------------------------------
- Operating System: `Darwin`
- Node Version:     `v16.11.0`
- Nuxt Version:     `0.0.0`
- Package Manager:  `unknown`
- Bundler:          `Webpack`
- User Config:      `-`
- Runtime Modules:  `-`
- Build Modules:    `-`
------------------------------

### Reproduction

Reproduction is unavailable. The bug is on bootstrap level.

### Describe the bug

When trying to create a new nuxt project with `npx nuxi init`  following error shown 

Nuxt CLI v3.0.0-27284664.232fdd8                                                                                                                        

 WARN  could not fetch remote https://github.com/nuxt/starter                                                                                                     

 WARN  Make sure you have installed git correctly

### Additional context

git installed - git version 2.26.0

 internet connection works

**Comment by danielroe** [comment]
see https://github.com/nuxt/framework/issues/1878 (with suggestions for solving).

**Comment by intermundos** [comment]
I don't understand why you close the issue. 

#1878 does not have any suggestion for solving the issue. 

I use no VPN, my git installation is OK and apparently internet connection is ok too. 

I tried resetting npm caches, reinstalling the nuxi, different node versions and still get the same error. 

Would be lovely to see you helping instead of closing the issue straight away.

**Comment by danielroe** [comment]
@intermundos Would you share some more details about your debugging so far?

**Comment by pi0** [comment]
Moving it to discussion as probably a setup issue

**Comment by nyxb** [comment]
Hey same error here my git works fine in vscode etc.
i havent VPN and normal start with npx nuxi init any-project works fine.

if i start it with npx nuxi init -t any-project to use the vercel option it doesnt work and generate this error
 WARN  Make sure you have installed git correctly

my git version is git version 2.35.1.windows.2

that prob is only on windows mashine

**Reply by danielroe** [reply]
It should be `npx nuxi init any-project`

**Reply by nyxb** [reply]
> It should be `npx nuxi init any-project`

And why write that in a manual?

![vercel](https://user-images.githubusercontent.com/98586665/157063651-d0dc18e3-8a36-42e4-9e9b-14478fbed9c3.png)

Or I am just too stupid 🙈

**Reply by danielroe** [reply]
The `-t v3-vercel` selects the `v3-vercel` template. If you want to add a project name, you can do `npx nuxi init -t v3-vercel my-project-name`.

**Reply by nyxb** [reply]
> The `-t v3-vercel` selects the `v3-vercel` template. If you want to add a project name, you can do `npx nuxi init -t v3-vercel my-project-name`.

Ok funny that's exactly what I thought after my previous sentence. I was just too stupid to understand correctly xD nvm

**Comment by captain828** [comment]
Getting the same error.
Followed https://v3.nuxtjs.org/getting-started/quick-start/

Git is fine.
No VPN either.

Not sure how to debug.

```
PS D:\my-proj> npx nuxi init nuxt-app     
Nuxt CLI v3.0.0-rc.2                                                                                                                                                                                              21:42:29

 WARN  could not fetch remote https://github.com/nuxt/starter                                                                                                                                                     21:42:29


 WARN  Make sure you have installed git correctly                                                                                                                                                                 21:42:29

PS D:\my-proj> git --version
git version 2.36.0.windows.1
```

**Reply by captain828** [reply]
This could be an issue with the latest `tiged` on windows but can't confirm rn.

**Comment by Rainer-Yu** [comment]
After I have reported errors many times, it will occasionally be useful once or twice. Personally, I think it may be a git problem.
![image](https://user-images.githubusercontent.com/62528887/168769189-b77b52b7-0d59-4dda-b5d2-a0581efb9f3b.png)
![image](https://user-images.githubusercontent.com/62528887/168769244-29170204-60c8-4f18-b627-81fd0b369511.png)

**Comment by quartzar** [comment]
I have the same issue. No idea how to solve.
![image](https://user-images.githubusercontent.com/79379455/170255353-b345a671-dffe-4aa9-b6f5-09a36a787370.png)

**Comment by TZZack** [comment]
A temporary solution:
Fisrt at all, after read the source code of init command, I saw that the init command will just clone the nuxt/starter repo, and all the files of this repo are the template of the nuxt3 and they will be placed directly on your project folder.
Therefore, the temporary solution is that we can download zip of the nuxt/starter repo(branch v3), and then place all the files to your project folder so that we can skip the init step.
At last, install the dependencies and go on normally.
I have tried successfully.

**Comment by kissu** [comment]
Related SO question/answer: https://stackoverflow.com/a/73108317/8816585

**Comment by HenrijsS** [comment]
Can you fetch the starter template from git?
```zsh
git clone https://github.com/nuxt/starter.git
```
